app/src/main/java/com/nmc/android/ui/LauncherActivity.kt

```diff
@@ -0,0 +1,64 @@
+package com.nmc.android.ui
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
+import com.nextcloud.client.preferences.AppPreferences
+import com.owncloud.android.R
+import com.owncloud.android.authentication.AuthenticatorActivity
+import com.owncloud.android.databinding.ActivitySplashBinding
+import com.owncloud.android.ui.activity.BaseActivity
+import com.owncloud.android.ui.activity.FileDisplayActivity
+import com.owncloud.android.utils.StringUtils
+import javax.inject.Inject
+
+class LauncherActivity : BaseActivity() {
+
+    companion object {
+        const val SPLASH_DURATION = 1500L
+    }
+
+    private lateinit var binding : ActivitySplashBinding
+
+    @Inject
+    lateinit var appPreferences: AppPreferences
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        //Mandatory to call this before super method to show system launch screen for api level 31+
+        installSplashScreen()
+
+        //flag to avoid redirection to AuthenticatorActivity
+        //as we need this activity to be shown
+        //Note: Should be kept before super() method
+        enableAccountHandling = false
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySplashBinding.inflate(layoutInflater)
+
+        setContentView(binding.root)
+        setSplashTitle()
+        scheduleSplashScreen()
+    }
+
+    private fun setSplashTitle() {
+        val appName = resources.getString(R.string.app_name)
+        val textToBold = resources.getString(R.string.project_name)
+        binding.tvSplash.text = StringUtils.makeTextBold(appName, textToBold)
```

Comment:
Instead of this, I think it is easier to understand/maintain to have two TextViews next to each other:
"bold""not bold" -> "MAGENTA""cloud"
call it
- splashScreenBold
- splashScreenNormal

and during set up check if splashScreenBold is empty, then hide it.
Those two strings can then be in setup.xml

Reply:
![image](https://user-images.githubusercontent.com/5836855/233085644-a9115e1b-30af-4e64-83cb-32f657342f6b.png)

I pushed a quick commit. Let me know what you think.